tests/python/pytorch/graphbolt/test_minibatch.py

```diff
@@ -889,6 +889,14 @@ def test_to_pyg_data():
     assert pyg_data.batch_size == expected_batch_size
     assert torch.equal(pyg_data.n_id, expected_n_id)
 
+    # Test with seeds in minibatch.
```

Comment:
Consider make it a separate test case, such that when clean up in the future it is easier?